nav_msgs/Path path
---
bool result
